lib/get-latest-ticket-numbers.rb

```diff
@@ -1,39 +1,32 @@
 require 'zendesk_api'
 
-@client = ZendeskAPI::Client.new do |config|
-  config.url = ENV['ZENDESK_URL']
-  config.username = ENV['ZENDESK_USER_EMAIL']
-  config.password = ENV['ZENDESK_USER_PASSWORD']
-
-  config.retry = true
-end
+require_relative 'zendesk-setup.rb'
 
 # year-364 days means we will delete 1 xtra day and be compliant for 23:59:59h
 # but then we will run again in 24h time
 
-today = Date.today.prev_day
-lastyear = today - 365
+today = Date.today
+lastyear = Date.today.prev_day - 365
 
 @latest_tickets = []
 
 #
 puts "Total Tickets 2018 to 1 year ago today"
 #
 
-puts @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>2018-01-01 updated_at<#{lastyear}").count
+puts @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>2018-01-01 updated_at<#{lastyear}").count.to_i
 
-ticket_count_for_year = @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>2018-01-01 updated_at<#{lastyear}").count
+ticket_count_for_year = @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>2018-01-01 updated_at<#{lastyear}").count.to_i
 # The Zendesk API has 100 tickets per page, so programatically
 # determine how many pages we have by rounding to the nearest 100.
 number_of_pages = (ticket_count_for_year.to_f / 100).ceil + 1
 
 (1..number_of_pages).each do |i|
-  # We use updated_at>2018 because we have removed ALL tickets before this date previously.
   @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>2018-01-01 updated_at<#{lastyear}").page(i).each do |ticket|
```

Comment:
Can we update the query to include 1st January?

```suggestion
  @client.search(:query => "type:ticket group_id:20188163 status:closed updated_at>=2018-01-01 updated_at<#{lastyear}").page(i).each do |ticket|
```